docs/SLACK_BOT.md

```diff
@@ -0,0 +1,20 @@
+## How to setup the Slack Bot?
```

Comment:
This is a separate doc, heading should have just one #

Reply:
Usage goes under it though. Is that correct?

Reply:
Updated as requested.